src/StdinFileInfo.php

```diff
@@ -23,6 +23,7 @@ final class StdinFileInfo extends \SplFileInfo
 {
     public function __construct()
     {
+        parent::__construct('');
```

Comment:
Does it actually fix something or is it here just to satisfy PHPStan? I prefer to ignore errors if there's no real problem reported.

Reply:
It is missing a call to the parent constructor, that would set path value.

Explicit over implicit - better call it to set it than not set it and cross-fingers nothing will break.